test/parallel/test-process-cpuUsage.js

```diff
@@ -33,27 +33,57 @@ for (let i = 0; i < 10; i++) {
   assert(diffUsage.system >= 0);
 }
 
+const invalidUserArgument = new RegExp('^TypeError: value of user ' +
```

Comment:
@edsadr it would be great to have it documented, I'm not sure where, maybe in the test-writing guide for now? I think the easiest way to get started would be to raise a PR with whatever you think it should be, @mention nodejs/collaborators, and see what the consensus is.